src/platform-includes/distributed-tracing/how-to-use/react-native.mdx

```diff
@@ -0,0 +1,13 @@
+If you're using the current version of our React Native SDK distributed tracing will work out of the box.
+
+```js
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracePropagationTargets: [
+    "https://myproject.org",
+    "https://.*.otherservice.org/.*",
+  ],
+});
+```
+
+If you're using version `5.9.x` or below, you'll need to have our <PlatformLink to="/performance/">performance monitoring feature enabled</PlatformLink> in order for distributed tracing to work.
```

Comment:
Shouldn't this be `5.8.x` or below? The release notes for sentry-react-native 5.9.0 say you don't need perf monitoring